### DISCUSSION TITLE
Phone call and Notification blocker for android

### DISCUSSION THREAD

**Discussion by adityadi98** [NativeScript/NativeScript#9642] [Q&A]
**Is your feature request related to a problem? Please describe.**
Yes, Block incomming phone calls for all or some specific numbers...

**Comment by Ross-Olivares** [comment]
Hi @adityadi98 
Can you tell us bit more about what you are trying to do, if you can send us a sample project that can reproduce the issue.
Let us know.

**Comment by adityadi98** [comment]
Thanks for a reply @Ross-Olivares ,
Actually I am working on a Exam Application where on a certain screen (during examination) the user (student) can't receive any incomming phone call neither notifications from other apps.. as there are solutions to block calls and notifications in android java can't find any for Nativescript Angular 
also tried [https://www.nsplugins.com/plugin/nativescript-callblock](url) but not able to do it in angular...
